src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -281,17 +292,37 @@ private static void GetTextResourceForLanguage(Dictionary<string, Dictionary<str
             return jsonTexts;
         }
 
+        /// <summary>
+        /// Get the markdown text specific for a key identified in the filename.
+        /// </summary>
+        /// <param name="markdownFileName">Filename for file with markdown text for a key</param>
+        /// <returns>Markdown text as a string</returns>
+        public async Task<string> GetTextMarkdown(string markdownFileName)
+        {
+            var textsFileRelativeFilePath = Path.Combine(CONFIG_FOLDER_PATH, LANGUAGE_RESOURCE_FOLDER_NAME, MARKDOWN_TEXTS_FOLDER_NAME, markdownFileName);
```

Comment:
No biggie, but this combination of folders seem to repeat it self many places - you could consider creating a small private method eg. GetTextMarkdownPath(markdownFileName). That way you have one place where you have to remember to do all the Path.Combine.